bundle-workflow/test.sh

```diff
@@ -9,4 +9,4 @@
 set -e
 
 DIR="$(dirname "$0")"
-"$DIR/run.sh" "$DIR/src/test.py" $@
+"$DIR/run.sh" "$DIR/src/test.py" $@
```

Comment:
Did you delete src/test.py?

Reply:
Yes, @setiah will add the test.sh in #456 as part of his PR. 
Ref: https://github.com/opensearch-project/opensearch-build/pull/423#discussion_r708765944